packages/shared-components/src/formio-overrides/wizard-overrides.js

```diff
@@ -140,12 +137,38 @@ Wizard.prototype.attachHeader = function () {
     });
   }
 
+  // Copy of nextPage() from formio.js/src/Wizard.js, but without emitting event when going to next page
   const validateAndGoToNextPage = () => {
+    if (this.options.readOnly) {
+      return this.beforePage(true).then(() => {
+        return this.setPage(this.getNextPage());
+      });
+    }
+
+    // Validate the form, before go to the next page
+    if (this.checkValidity(this.localData, true, this.localData, true)) {
+      this.checkData(this.submission.data);
+      return this.beforePage(true).then(() => {
+        return this.setPage(this.getNextPage()).then(() => {
+          if (!(this.options.readOnly || this.editMode) && this.enabledIndex < this.page) {
+            this.enabledIndex = this.page;
+            this.redraw();
+          }
+        });
+      });
+    } else {
+      this.currentPage.components.forEach((comp) => comp.setPristine(false));
+      this.scrollIntoView(this.element);
+      return Promise.reject(this.showErrors([], true));
+    }
+  };
+
+  const validateUntilLastPage = () => {
     if (this.isLastPage()) {
       this.emit("submitButton"); // Validate entire form and go to summary page
     } else {
-      this.nextPage() // Use "nextPage" function, which validates current step and moves to next step if valid or display errors if invalid
```

Comment:
Kan man skille på eventen, i stedet for å kopiere fra form.io? F.eks med en ekstra prop